lib/csp.js

```diff
@@ -79,16 +79,16 @@ function getCspNonce (req, res) {
 }
 
 function addUpgradeUnsafeRequestsOptionTo (directives) {
-  if (config.csp.upgradeInsecureRequests === 'auto' && config.useSSL) {
+  if (config.get('csp').upgradeInsecureRequests === 'auto' && config.get('useSSL')) {
```

Comment:
can this be `.get('csp.upgradeInsecureRequests')`?

Reply:
Both works. the regex I used to transform those, was focused on the first option.

https://github.com/mozilla/node-convict/#configgetname